src/frequenz/sdk/timeseries/_formula_engine/_formula_steps.py

```diff
@@ -207,6 +207,14 @@ def __init__(self, fetchers: List[MetricFetcher[QuantityT]]) -> None:
         """
         self._fetchers: list[MetricFetcher[QuantityT]] = fetchers
 
+    def fetchers(self) -> list[MetricFetcher[QuantityT]]:
```

Comment:
Thanks. I am not yet sure we will keep that code. We can leave it in, for sure (I was wondering why it was missing). I was trying to access somehow the type information about a fetcher here. Not yet done nor succeeded though :)

Reply:
I would remove it or make it a property to follow the pattern of other similar methods that are just accessing an attribute (effectively making it read-only to the user). Actually I would probably return `Sequence[...]` instead of `list` so it's a bit more read-only (the type checker won't allow users to mutate the returned `list` if you use `Sequence`.